docs/content/latest/secure/tls-encryption/connect-to-cluster.md

```diff
@@ -15,30 +15,22 @@ isTocNested: true
 showAsideToc: true
 ---
 
-To connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-server encryption is enabled, you need to generate client certificate files that enable the client to connect to the YugabyteDB cluster.
+You can connect CLIs, tools, and APIs to a remote YugabyteDB cluster when client-to-server encryption is enabled.
 
 ## Prerequisites
 
-Before you can use client certificates to connect to your YugabyteDB clusters, server-to-server encryption and client-to-server encryption must be enabled. For details, see [Enable server-to-server encryption](../server-to-server/) and [Enable client-to-server encryption](../client-to-server).
+You do not need client certificates in order to connect to your YugabyteDB clusters, but you have to enable server-to-server encryption and client-to-server encryption. For more information, see [Enable server-to-server encryption](../server-to-server/) and [Enable client-to-server encryption](../client-to-server).
 
-Also, you need to create the required client certificates. For details, see [Create client certificates](../client-certificates).
+Each client that connects to a YugabyteDB cluster needs the following file to be accessible on the client computer:
 
-Each client that connects to a YugabyteDB cluster needs the following files to be accessible on the client computer.
+- `ca.crt` — root certificate file (YSQL and YCQL). See [Generate the root certificate file](../server-certificates/#generate-the-root-certificate-file) for instructions on how to generate this file.
 
-- `ca.crt` — root certificate file (YSQL and YCQL)
-  - To generate, see [Generate the root certificate file](../server-certificates/#generate-the-root-certificate-file)
-- `yugabytedb.crt` — private node certificate (YSQL only)
-  - To generate, see [Generate client private key and certificate](../client-certificates/#generate-private-key-and-certificate)
-- `yugabytedb.key` — private node key (YSQL only)
-  - To generate, see [Generate client private key and certificate](../client-certificates/#generate-private-key-and-certificate)
+  This file should be available in the `~/.yugabytedb`, the default location for TLS certificates when running the YSQL shell (`ysqlsh`) locally.
 
-All three files should be available in the `~/.yugabytedb`, the default location for TLS certificates when running the YSQL shell (`ysqlsh`) locally.
+## Connecting to a YugabyteDB Cluster
 
-## Connect to a YugabyteDB cluster
+For each client, the steps assume that you have performed the following:
 
-For each of the clients below, the steps assume that you have:
-
-- Added the required client certificates to the `~/.yugabytedb` directory (or a directory specified using the `--certs_for_clients_dir` flag). For details, see [Create client certificates](../client-certificates).
```

Comment:
we still might need the CA cert in the .yugabytedb folder, depending on the sslmode used in the client. I can try this scenario and get back to you.

Reply:
@iSignal I'm no longer sure I understand the ask. I thought the ticket was about removing any content related to client certs because the users didn't need to install them anymore. Consequently, I removed the "Create client certs" file and we wouldn't be able to link to it.

Reply:
My comment was about the CA cert (cert authority) not the client certs themselves. It was addressed by your last revision which asked for ca.crt to be placed in .yugabytedb